shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformChannel.java

```diff
@@ -517,7 +517,7 @@ public interface PlatformMessageHandler {
     void setSystemUiOverlayStyle(@NonNull SystemChromeStyle systemUiOverlayStyle);
 
     /** The Flutter application would or would not like to handle navigation pop events itself. */
-    void setFrameworkHandlesBack(boolean frameworkHandlesBack);
+    default void setFrameworkHandlesBack(boolean frameworkHandlesBack) {}
```

Comment:
This makes sense to me since it is only used in a particular place. I'd suggest expanding the doc to mention where exactly this is relevant currently (`FlutterActivity`), just so it's clear we don't expect it to work or be used elsewhere.